server/item/item.go

```diff
@@ -107,6 +108,12 @@ type Weapon interface {
 	AttackDamage() float64
 }
 
+// CooledDown represents an item that has a cool down.
+type CooledDown interface {
+	// CoolDown is the duration of the cool down.
+	CoolDown() time.Duration
```

Comment:
Would name this `Cooldown`. `CoolDown` implies `cool down` which is a verb. Make sure you also change the `cool down` to `cooldown` where it is used as a noun.